sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java

```diff
@@ -228,13 +228,11 @@ public static synchronized FlitchioController getInstance(Context context) {
     /**
      * Check the installation of Flitchio Manager by comparing the version codes.
      *
-     * @throws FlitchioManagerDependencyException
+     * @return True if the Manager installed on the phone can be used by this version of the SDK.
      */
-    private static void checkFlitchioInstall(Context context)
-            throws FlitchioManagerDependencyException {
-        if (!BuildConfig.DEBUG && getFlitchioManagerVersionCode(context) < getVersionCode()) {
-            throw new FlitchioManagerDependencyException();
-        }
+    private static boolean isFlitchioManagerUsable(Context context) {
+        return BuildConfig.DEBUG // In debug, we consider the Manager always accessible
```

Comment:
I would prefer this comment to explain the why and not the what because I don't really understand why we do this in debug,